crates/hir/src/lib.rs

```diff
@@ -3190,6 +3190,14 @@ impl Type {
         matches!(self.ty.kind(Interner), TyKind::Raw(..))
     }
 
+    pub fn as_raw_ptr_ty(&self) -> Option<Type> {
```

Comment:
let's called it `remove_raw_ptr` to bring it inline with the `remove_ref` function

Reply:
makes sense.